### DISCUSSION TITLE
AsusFX504 - 0.8.3

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#15] [General]
### News / Fixes:
- Updated all config.plist parameters for OC 0.8.3, all OpenCore files and Kexts are also updated.
- Deleted: `agdpmod=vit9696` Boot Attribute and replaced with `agdpmod` property on iGPU Properties as a permanent setting.
- Removed EFI for Audio/HDMI variants (now the RELEASE is prioritizing audio, if your HDMI is not working correctly just move to "Delete" this tree: `PciRoot(0x0)/Pci(0x1F,0x3)` on DeviceProperties
- Updated DEBUG version too, so both (RELEASE and DEBUG) variants are updated to OC 0.8.3

### Known Issues:
- HDMI may not work great on certain displays especially if connected to a High Res Display

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore/releases/tag/0.8.3'>AsusFX504 - 0.8.3</a>.</em>

**Comment by Dv1101** [comment]
is there any fix for Audio  jack? I tried it on my TUF FX504GD (i5-8300h, uhd 630). 3.5mm audio jack working but only vocals are coming, Voice isnt coming. I tried different headphones too. In Bluetooth headphones sound is fine af. BTW **_Thanks for making perfect configuration of kexts_**. earlier i booted on my own configs but lots of missing there.

**Reply by RobyRew** [reply]
I recommend you try all possible audio layouts for the codec, the current one is `30` Decimal or `1E0000` on Hexadecimal 

Vendor | Codec | Revisions and layouts | MinKernel | MaxKernel
-- | -- | -- | -- | --
Realtek | ALC255/ALC3234 | 0x100002,layout 3, 11, 12, 13, 15, 17, 18, 20, 21, 27, 28, 29, 30, 31, 66, 71, 82, 86, 96, 99, 100, 255 | 13 (10.9) | —

If you find one that works better for you please respond here

**Reply by Dv1101** [reply]
Thanks, it was my fault. I forgot to Enable Connect drivers under audio section inside EFI. Layout 30 is working fine for FX504GD. _**All things works as mentioned for FX504GD.**_

**Comment by MyHoloLove** [comment]
I just want to thank you for your hard work. if not for you i would have to buy a mac which would cost me alot. it helped me a lot especially when i needed to use software's fow work unique to mac os. thanks to you i am able to use them. keep up the good work. your builds are super stable

**Reply by RobyRew** [reply]
Wow man, im glad is working great for you, I hope it continues to do!!
Take care!❤️

**Comment by MyHoloLove** [comment]
Hi,
I wanted to ask something. I updated to Mac OS Venture beta 3. it was successful. however whenever it boots up into the OS it auto restarts after exactly 30 sec every time. i wanted to know if there is something i need to change in the config.plist for this?

although i restored mac os Monterey for now i just wanted to know what needs to be changes

Thanks

**Reply by RobyRew** [reply]
It may be because of Airportitlwm.kext
Te current version that is provided in this RELEASE is for Monterrey, you may want to try the version for Ventura.
Tell me if you managed to have it working.
I hope to have Some time left so I can update to Ventura too.

Cheers!

**Reply by MyHoloLove** [reply]
Thanks, I'll let you know if it works

**Reply by MyHoloLove** [reply]
IT WORKS!!

replacing the Airportitlwm.kext was the right solution. i tested to see if anything was not working. only HDMI & dGPU are not working rest what you mentioned in the official description all of that is working and you will need to generate SMBIOS again mine was not not logging into face time & Messages after replacing kext & editing the .plist. after generating the SMBIOS again all that started working.

Kext : https://github.com/OpenIntelWireless/itlwm/releases/tag/v2.2.0-alpha   (Debug version)

![Screenshot 2022-08-18 at 5 10 45 PM](https://user-images.githubusercontent.com/65754586/185387749-0ce76bfc-3e4c-4d7f-9236-a5267ca176b2.png)

**Reply by RobyRew** [reply]
Im soo happy, thanks for the feedback!!

I'll consider your info useful for wen Im going to update to Ventura.

Thanks a lot!

**Reply by Dv1101** [reply]
Still not logging in after generating smbios again. any fix or clue for it?

**Reply by RobyRew** [reply]
> 
> 
> Still not logging in after generating smbios again. any fix or clue for it?
> 
> 

hello, I have no idea why, but what i can tell you is that in the past I had real a real mac and maybe that's why my Apple ID is not blocked on hackintoshes, but this is a theory of mine, because I have no idea what's happening to other builds of mine or other people

**Reply by MyHoloLove** [reply]
> Still not logging in after generating smbios again. any fix or clue for it?

I have a question did u by any chance ever try to login without generatieing any SMBIOS?
== if u did, that ID is probably blocked from logging in you need to do a clean install with new SMBIOS and make a new apple account and make sure u log it in a different SMBIOS.

**Reply by Dv1101** [reply]
> > Still not logging in after generating smbios again. any fix or clue for it?
> 
> I have a question did u by any chance ever try to login without generatieing any SMBIOS? == if u did, that ID is probably blocked from logging in you need to do a clean install with new SMBIOS and make a new apple account and make sure u log it in a different SMBIOS.

its same for new apple id. i generated new sm bios and created new apple id too. still it's not logging in. although i'm not using imessage and facetime so no worries for it.

**Comment by code4power** [comment]
Hello everyone and thank you very much for the work done Mr. "RobyRew".

I have a problem now, I updated to 12.6 and as soon as I enter the system, the apple appears, and at about 40% it stays for a while and the apple turns a little red and then restarts the laptop in recovery mode. Could you help me a little what I could do? 
Thank you.
=== FX504 GM ===

**Reply by MyHoloLove** [reply]
this has happened to me once before when i used OC 0.7.8 for mac os Monterey 12.5.1 this issues was fixed by upgrading my efi to the latest open core version. i suggest u downgrade for now and keep an eye on r/hackintosh on reddit as someone will post about this issue and its solution.

**Reply by RobyRew** [reply]
> Hello everyone
Could you help me a little what I could do? 
> 
> Thank you.
> === FX504 GM ===

I'll update the EFI on 0.8.5 (next month) and also update to macOs Ventura, for now i suggest you to do what mr. @MyHoloLove said maybe it fixes your problem...

respond if you fixed ;)

**Comment by code4power** [comment]
@RobyRew .... 
Thank you very much for your work.
If you have time in the future to solve a small problem or maybe it is solved and I do not know how to do ...

When I count my mouse or headphones via Bluetooth and the laptop enters Sleep (I think), nothing works through Bluetooth until I restart.
=== FX504 GM===

And tell me if I have the opportunity to make you donations, and where?
Thank you again!

**Reply by RobyRew** [reply]
> When I count my mouse or headphones via Bluetooth and the laptop enters Sleep (I think), nothing works through Bluetooth until I restart. === FX504 GM===

I think I know what might be a fix for that, ill look into it ASAP because im a little busy with school, I will not forget it ;) 

> And tell me if I have the opportunity to make you donations, and where? Thank you again!

It's not necessary to donate but if you really want to, here's my PayPal: https://www.paypal.com/paypalme/RobyRew
_You would also be the first person to give me a donation😜_

**Reply by code4power** [reply]
Salut! Esti RO?
am facut o mica donatie, dar iti mai fac, spor la treaba! pe viitor imi este mai usor pe Revolut ;)

**Reply by RobyRew** [reply]
> Salut! Esti RO?
> 
> am facut o mica donatie, dar iti mai fac, spor la treaba! pe viitor imi este mai usor pe Revolut ;)

Da sunt român, ma bucur mult ca mai sunt romani pe aici 😜, mai ales de ăștia de treaba ca tine!

Mulțumesc pentru donație, o sa ma uit sa vad cum e cu Revolut, am văzut publicitate acum nu mult timp dar nu am căutat încă, o sa anunț când voi face, iarăși îți mulțumesc si îți doresc un sfârșit de săptămana plăcut! ;)

**Reply by RobyRew** [reply]
> Salut! Esti RO?
> am facut o mica donatie, dar iti mai fac, spor la treaba! pe viitor imi este mai usor pe Revolut ;)

Salut, asi avea nevoie sa fac cont de Revolut, știi de vreo promoție sau ai vreo-un cod de invitație?

**Comment by Dv1101** [comment]
![Screen Shot 2022-09-22 at 1 36 49 PM](https://user-images.githubusercontent.com/86505071/191693373-4d7ddbf4-4232-4d7a-9b1a-ed8d7a542cbd.png)

Airplay is working flawlessely 1080p straming with audio and no lag or buffer in playback. only continuity feature not working so far...

**Reply by MyHoloLove** [reply]
The your name wallpaper is he highlight here.

**Comment by Dv1101** [comment]
@RobyRew is there any way to increase overall font size or DPI scaling? on 1080p fonts is smaller than regular and in scale mode it increases content size with decreasing resolution.

**Reply by RobyRew** [reply]
> @RobyRew is there any way to increase overall font size or DPI scaling? on 1080p fonts is smaller than regular and in scale mode it increases content size with decreasing resolution. 

ther's a tool called TinkerTool is free and gives you the possibility to play wit a lot of hidden macos features (some of them about fonts)![image](https://user-images.githubusercontent.com/49403384/192102042-236e1274-bb01-455e-8acd-02083d8bb881.jpeg)
[TinkerTool Download page](http://www.bresink.com/osx/0TinkerTool/download.php)

**Reply by Dv1101** [reply]
Thanks 👍🏻

**Comment by code4power** [comment]
It boots me a little slowly, and with another "EFI" it cutes me much faster. You can help me with some solutions, maybe I was wrong with something ... Thank you.

**Comment by RobyRew** [comment]
> another "Empi" it cutes me much faster. You can help me with some solutions

what's that "Empi"?

> maybe I was wrong with something ...

what happened?

**Reply by code4power** [reply]
EFI :D
